packages/api/endpoints/rules.js

```diff
@@ -60,26 +64,29 @@ async function get(req, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function post(req, res) {
+  const {
+    model = new models.Rule(),
+    dbClient = await getKnexClient(),
+  } = req.testContext || {};
+
+  let record;
+  const rule = req.body || {};
+  const name = rule.name;
+  rule.createdAt = Date.now();
+  rule.updatedAt = Date.now();
```

Comment:
It seems kind of unnecessary to set these properties directly on the rule if they're going to be reset in the `model.create` call

https://github.com/nasa/cumulus/pull/2012/files#diff-2a37aa7b44507f6480bc0edc75d3aa7e70031468d98c108f8aef958a2098ef91R245

So I would just pass in the right values in the call to model.create rather than setting on the incoming rule.object

Reply:
[`9e37ed6` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/9e37ed6f9ae36d213d879fe88fcaab071d440c51)